UPGRADING.md

```diff
@@ -5,6 +5,15 @@ This document will be updated to list important BC breaks and behavioral changes
 
 ### upgrading to 2.0.0
 
+ * The SensioFrameworkExtraBundle `@View` annotation must be enabled to use the `ViewResponseListener`:
```

Comment:
In fact that's the SensioFrameworkExtraBundle ``@Template`` annotation. 
What about replacing it by ``the SensioFrameworkExtraBundle view annotations`?